src/services/dashboards/dashboard-customize/DashboardCustomizePage.vue

```diff
@@ -114,6 +116,7 @@ const variableState = reactive({
     variableData: computed(() => dashboardDetailState.variables),
     variableProperties: computed(() => dashboardDetailState.variablesSchema.properties),
     order: computed(() => dashboardDetailState.variablesSchema.order),
+    referenceType: computed(() => store.getters['reference/allReferenceTypeInfo']),
```

Comment:
suggestion: I recommend to use this name :)

```suggestion
    allReferenceTypeInfo: computed(() => store.getters['reference/allReferenceTypeInfo']),
```